kodilla-collections-advanced/src/main/java/com/kodilla/collections/adv/immutable/homework/Task.java

```diff
@@ -0,0 +1,20 @@
+package com.kodilla.collections.adv.immutable.homework;
+
+public class Task {
+
+    protected String title;
+    protected int duration;
+
+    public Task(String title, int duration) {
+        this.title = title;
+        this.duration = duration;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public int getDuration() {
+        return duration;
+    }
+}
```

Comment:
public class MutableTask extends Task {

    public MutableTask(String title, int duration) {
        super(title, duration);
    }
   
   public void setTitle(String title) {
       this.title = title;
   }

    public String getTitle() {
        return title;
    }

    public int getDuration() {
        return duration;
    }
}